src/components/List/Item.jsx

```diff
@@ -0,0 +1,57 @@
+import { useState } from 'react';
+import styles from './item.css';
+
+export default function Item({ item, updateHandler, deleteHandler }) {
+  const [isEditing, setIsEditing] = useState(false);
+  const [name, setName] = useState(item.name);
+  return isEditing ? (
+    <div aria-label={`${item.name}`}>
```

Comment:
Nice work implementing the optional chaining of what is being rendered to the page here.